### PR TITLE
Add ppci

### PR DESCRIPTION
**Pull request by windelbouwman**

### CODE REVIEW

File: recipes/ppci/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
```
Comment (synapticarbors) [inline recipes/ppci/meta.yaml]: Does this package support python 2? The metadata in the `setup.py` only mentions python 3. Looking at `ppci / __init__.py` it looks like it's python 3 only. If this is the case, you need to drop the `noarch: python` and add `skip: True  # [py2k]`

File: recipes/ppci/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+test:
+  imports:
+    - ppci
+  commands:
+    - ppci-build.py -h
+
+about:
+  home: http://ppci.readthedocs.io/
+  license: BSD-2-Clause
```
Comment (synapticarbors) [inline recipes/ppci/meta.yaml]: Add `license_file` manually:

https://conda-forge.org/docs/meta.html#packaging-the-license-manually

File: recipes/ppci/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
```
Comment (synapticarbors) [inline recipes/ppci/meta.yaml]: Add imports of submodules. For example:

```
- ppci.arch
- ppci.codegen
...
etc
```

File: recipes/ppci/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
```
Comment (synapticarbors) [inline recipes/ppci/meta.yaml]: looks like you're missing run time dependencies. Error message for import failure for `colorama`. There may be others, but I'll leave it to you to find the dependencies.

File: recipes/ppci/meta.yaml (hunk 1 above)
Comment (windelbouwman) [reply recipes/ppci/meta.yaml]: Yes, it is python 3 only.

File: recipes/ppci/meta.yaml (hunk 4 above)
Comment (windelbouwman) [reply recipes/ppci/meta.yaml]: Okay, I added the colorama dependency.

File: recipes/ppci/meta.yaml (hunk 3 above)
Comment (windelbouwman) [reply recipes/ppci/meta.yaml]: Okay, good idea!

File: recipes/ppci/meta.yaml (hunk 2 above)
Comment (windelbouwman) [reply recipes/ppci/meta.yaml]: Yes, done.

**Comment by synapticarbors** [comment]
@windelbouwman -- Thanks for your contribution to conda-forge. A feedstock for this recipe should be generated soon and you'll be receiving several automated messages about it. When the feedstock has been rendered, please take a look at the README in that repo for information about updating and maintaining the recipe. One of the key points is all updates should be done by forking that repo and following a standard pull request workflow. If you haven't looked at it already the docs are quite helpful:

https://conda-forge.org/docs/

If you run into any difficulties please let us know.

**Pull request merged**